tests/build_stitched_index.cpp

```diff
@@ -0,0 +1,855 @@
+// author: Siddharth Gollapudi
```

Comment:
Contributions are indicated by commit author. Please replace this header with Microsoft copyright and license info